PromoExchange/batman
# Fix upcharge open range

https://trello.com/c/IMkzK68S
### Test Plan
1. Login to preview site -  http://px-batman-stage-pr-703.herokuapp.com/company_store/pimco
2. Select a yeti mug, get a quote for 222
3. Login to stage pimco site - http://pimco-stage.promox.co/company_store/pimco
4. Select a yeti mug, get a quote for 222
5. Ensure preview site price is $9,099.64 and stage site price is $8,751.46.

If you can find an easy way to get the rails log from the heroku preview app I would appreciate the insight. If you are able to you will see the supplier upcharge being applied in the prebid log.
